packages/jest-haste-map/src/crawlers/node.ts

```diff
@@ -139,10 +139,6 @@ export = function nodeCrawl(
   removedFiles: FileData;
   hasteMap: InternalHasteMap;
 }> {
-  if (options.mapper) {
```

Comment:
I think this makes sense, not sure why it'd need to throw. Maybe someone has a good reason, though 😀

Reply:
Yea. There are other options we ignore in Node crawler (like `computeSha1`). Maybe it was some kind of development helper that wasn't removed? Anyway, it's gone now :P